pyproject.toml

```diff
@@ -11,6 +11,13 @@ authors = [
     "Sophie Arana <sarana@turing.ac.uk>",
     "Eric Daub <edaub@turing.ac.uk>",
     "Steve Niederer <sniederer@turing.ac.uk>",
+    "Radka Jersakova <rjersakova@turing.ac.uk>",
+    "Edwin Brown <wbrown@turing.ac.uk>",
+    "Chris Sprague <csprague@turing.ac.uk>",
+    "Ed Chalstrey <echalstrey@turing.ac.uk>",
+    "Marjan Famili <mfamili@turing.ac.uk>",
+    "Paolo Conti <pconti@turing.ac.uk>",
+    "Sam Greenbury <sgreenbury@turing.ac.uk>",
```

Comment:
We should add Andrew here. I am also in favour of just removing all names and replacing with something like "The AutoEmulate team" and rely on the fact that we capture all contributions in the README already and so don't have to worry about doing it here, especially if the project keeps growing (hopefully). Maybe we should decide what's better as a team?

Reply:
I agree, annoying to update in multiple places